css/style.css

```diff
@@ -0,0 +1,155 @@
+.header {
+    font-weight: bold;
+    padding-left: 10px;
+    padding-top: 5px;
+    padding-bottom: 5px;
+    background: #07889B;
+    color: #FFF;
+}
+
+.title {
+    font-size: 30px;
+}
+
+.inputSection {
+    padding: 10px;
+    width: calc(100% - 17em - 10px);
+    margin: 0;
+}
+
+.tableContainer {
+    display: inline-block;
+    padding-right: 0.5em;
+    position: relative;
+    vertical-align: top;
+}
+
+.table {
+    width: auto;
+    margin-bottom: auto;
+    background-color: transparent;
+    border-spacing: 0;
+    border-collapse: collapse;
+}
+
+
+.table td {
+    vertical-align: middle;
+    padding: 5px 0 5px 3px;
+    border-top: inherit;
+}
+
+.table-bordered td,
+.table-bordered th {
+    border: 1px solid #ddd;
+}
+
+.table th {
+    padding: 8px;
+    line-height: 1.42857143;
+    vertical-align: top;
+    text-align: left;
+}
+
+.table .name input {
+    width: 10em;
+}
+
+.section .borderLine {
+    border-bottom: 1px solid #66B9BF;
+    border-top: 1px solid #B2DCDF;
+    margin-bottom: 15px;
+    margin-top: 15px;
+}
+
+.section label {
+    position: relative;
+    margin-top: 1.7em;
+}
+
+.section .tableTitle {
+    padding-left: 20px;
+    font-weight: bold;
+}
+
+.section .total {
+    padding-left: 10em;
+}
+
+.section .buttonContainer {
+    width: 30px;
+}
+
+.section .btn {
+    font-weight: bold;
+    opacity: 0;
+    transition: opacity .35s ease;
+}
+
+.btn-danger {
```

Comment:
we can probably rename these to something more specific to our use, like "delete"